bentoml/_internal/frameworks/xgboost.py

```diff
@@ -116,10 +119,10 @@ def load(
 @inject
 def save(
     name: str,
-    model: "xgb.core.Booster",
+    model: "xgb.Booster",
     *,
-    booster_params: t.Optional[t.Dict[str, t.Union[str, int]]] = None,
-    metadata: t.Optional[t.Dict[str, t.Any]] = None,
+    booster_params: t.Dict[str, t.Union[str, int]] = FrozenDict(),
```

Comment:
FrozenDict will break `_setup_booster_params` below, having default value for non primitive types is generally not recommended